Explorer/Assets/Scripts/Global/Dynamic/PortableExperiences/PortableExperiencesController.cs

```diff
@@ -0,0 +1,218 @@
+
+            if (!worldUrl.IsValidUrl()) throw new ArgumentException("Invalid Spawn params. Provide a URN or an ENS name");
+
+            var portableExperiencePath = URLDomain.FromString(worldUrl);
+            URLAddress url = portableExperiencePath.Append(new URLPath("/about"));
+
+            await UniTask.SwitchToMainThread();
+
+            GenericDownloadHandlerUtils.Adapter<GenericGetRequest, GenericGetArguments> genericGetRequest = webRequestController.GetAsync(new CommonArguments(url), ct, ReportCategory.REALM);
+
+            ServerAbout result = await genericGetRequest.OverwriteFromJsonAsync(serverAbout, WRJsonParser.Unity);
+
+            if (result.configurations.scenesUrn.Count == 0)
+            {
+                //The loaded realm does not have any fixed scene, so it cannot be loaded as a Portable Experience
+                throw new Exception($"Scene not Available in provided Portable Experience with ens:{ens} - urn: {urn}");
+            }
+
+            realmData.Reconfigure(
+                new IpfsRealm(web3IdentityCache, webRequestController, portableExperiencePath, result),
+                result.configurations.realmName.EnsureNotNull("Realm name not found"),
+                result.configurations.networkId,
+                result.comms?.adapter ?? string.Empty,
+                result.comms?.protocol ?? string.Empty
+            );
+
+            ISceneFacade parentScene = scenesCache.Scenes.FirstOrDefault(s => s.SceneStateProvider.IsCurrent);
+            string parentSceneName = parentScene != null ? parentScene.Info.Name : "main";
+            Entity portableExperienceEntity = world.Create(new PortableExperienceComponent(realmData, parentSceneName, isGlobalPortableExperience));
+            PortableExperienceEntities.Add(ens, portableExperienceEntity);
+
+            return new IPortableExperiencesApi.SpawnResponse
+                { name = realmData.RealmName, ens = ens, parent_cid = parentSceneName, pid = portableExperienceEntity.Id.ToString() };
+        }
+
+        public bool CanKillPortableExperience(string ens)
+        {
+            ISceneFacade currentSceneFacade = scenesCache.Scenes.FirstOrDefault(s => s.SceneStateProvider.IsCurrent);
+            if (currentSceneFacade == null) return false;
+
+            if (PortableExperienceEntities.TryGetValue(ens, out Entity portableExperienceEntity))
+            {
+                PortableExperienceComponent portableExperienceComponent = world.Get<PortableExperienceComponent>(portableExperienceEntity);
+                return portableExperienceComponent.ParentSceneId == currentSceneFacade.Info.Name;
+            }
+
+            return false;
+        }
+
+        public List<IPortableExperiencesApi.SpawnResponse> GetAllPortableExperiences()
+        {
+            spawnResponsesList.Clear();
+
+            foreach (var portableExperience in PortableExperienceEntities)
```

Comment:
You don't need a list to make a query to ECS: if you had a separate component to indicate PX you could filter by it in QUERY